cocinaborcelle/src/pages/api/updateConta.ts

```diff
@@ -0,0 +1,52 @@
+import { auth } from "@/../auth/lucia";
+import { NextApiRequest, NextApiResponse } from "next";
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+	if (req.method != "POST") {
```

Comment:
```suggestion
	if (req.method != "PATCH") {
```

para alterar dados de uma instância do banco de dados, pode utilizar o método patch. Padronizar os métodos http com o seu contexto de uso ajuda em depurações futuras

obs.: a exceção que curto utilizar para isso é o de login de usuário, que, apesar de pegar informações do banco de dados, não é top usar get e sim post, pois os métodos que alteram algo do banco de dados possuem um nível de segurança maior, como post, put, patch, enfim